packages/office-ui-fabric-react/src/common/_semanticColorVariables.scss

```diff
@@ -28,3 +28,58 @@ $selectedActiveBackgroundColor: $ms-color-neutralTertiaryAlt;
 $selectedActiveForegroundColor: $ms-color-neutralPrimary;
 
 $dividerColor: $ms-color-neutralLight;
+
```

Comment:
The semantic slots here are duplicated across multiple branches I'm working on, to make my life easier for later merges.